lib/ProductOpener/Ingredients.pm

```diff
@@ -4214,10 +4215,10 @@ my %prefixes_before_dash = (fr => ['demi', 'saint',],);
 my %ignore_phrases = (
 	de => [
 		'\d\d?\s?%\sFett\si(\.|,)\s?Tr(\.|,)?',    # 45 % Fett i.Tr.
-		"inklusive",
+		'inklusive',
 	],
-	en => ["na|n/a|not applicable",],
-	fr => ["non applicable|non concerné",],
+	en => ['contains 2% or less of:',],    # Contains 2% or less of (Xanthan Gum)
```

Comment:
We should keep the "contains 2% or less of:", it's an important part of the ingredients list. We don't parse it correctly yet, but we do not want to remove it from ingredients lists. Instead we can ignore it during parsing (or better, add support for it to set the percent_min property), and keep it in the list of ingredients.